### PR TITLE
py-xattr: fix a package that was pointing to a fork

### PR DESCRIPTION
**Pull request by alalazo**
closes #7136

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-xattr/package.py [hunk 1]
```diff
@@ -3,17 +3,16 @@
...
 
 class PyXattr(PythonPackage):
     """A python interface to access extended file attributes,
-        sans libattr dependency"""
+    sans libattr dependency.
+    """
 
     homepage = "http://pyxattr.k1024.org/"
-    git      = "https://github.com/fwang2/pyxattr.git"
+    git      = "https://github.com/iustin/pyxattr.git"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-xattr/package.py]: Is this package the same as https://pypi.org/project/xattr/? In that case, we could download tarballs straight from PyPI.

**Comment by fwang2** [comment]
I don't quite recall why, did this a while back. It is fine if you want to
change it back after sanity check.

thanks

File: var/spack/repos/builtin/packages/py-xattr/package.py (hunk 1 above)
Comment (alalazo) [reply var/spack/repos/builtin/packages/py-xattr/package.py]: Done

**Comment by alalazo** [comment]
@adamjstewart Being a low-traffic package I'll go on and merge this without waiting for a re-review. Feel free to write here if you have concerns and I'll address them in another PR.

**Pull request merged**